LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2014, Walmart and other contributors.
+Copyright (c) 2012-2015, Walmart and other contributors.
```

Comment:
Don't extend it. Instead, end the Walmart period and just leave the other contributors. E.g.:

```
Copyright (c) 2012-2014, Project contributors.
Copyright (c) 2012-2014, Walmart.
```